### PR TITLE
LanguageChoiceView Fixes

### PR DESCRIPTION
**Pull request by blakegreen32**
Ticket:  https://nanopay.atlassian.net/browse/NP-2669
Screen Shot 2020-11-12 at 4 15 12 PM

### CODE REVIEW

File: src/foam/nanos/auth/LanguageChoiceView.js [hunk 1]
```diff
@@ -183,7 +186,7 @@ foam.CLASS({
       let country = await this.countryDAO.find(language.variant)
       let label = language.variant != "" ? `${language.name}(${language.variant})` : `${language.name}`
       if ( country && country.name != null ) {
-        label = `${language.name}(${this.getCountryName(country.name, country.name)})`
+        label = `${language.name}\u00A0(${this.getCountryName(country.name, country.name)})`
```
Comment (kennyqiyenkan) [inline src/foam/nanos/auth/LanguageChoiceView.js:188]: Does this need unicode? A backtick should already detect a space.

**Review by kennyqiyenkan — CHANGES_REQUESTED** [review]
Could you provide a small screenshot as well? Better to see the changes

**Comment by blakegreen32** [comment]
Added screenshot

File: src/foam/nanos/auth/LanguageChoiceView.js (hunk 1 above)
Comment (blakegreen32) [reply src/foam/nanos/auth/LanguageChoiceView.js:188]: Xuerong had suggested that but it caused the string to break and overlap. After looking online I found that unicode and it worked fine.

**Review by kennyqiyenkan — APPROVED** [review]

**Pull request merged**